### PR TITLE
Add feature for changing console button tooltip to "Console in use"

### PR DESCRIPTION
**Pull request by bond95**

### CODE REVIEW

**Comment by matobet** [comment]
This change seems to be in direct conflict with #33 @bond95 do you intend to base one on the other?

File: src/components/VmActions/index.js [hunk 1]
```diff
@@ -107,11 +107,17 @@ const VmActions = ({ vm, isOnCard = false, onGetConsole, onShutdown, onRestart,
     cancelText: 'Cancel',
   }
 
+  let consoleStatus = 'Get default console'
+
+  if (vm.get('sessions') && vm.get('sessions').filter(c => c.get('consoleUser')).size) {
```
Comment (matobet) [inline src/components/VmActions/index.js]: please use `.isEmpty()` instead of relying on the `size` > 0 causing the condition to be true

**Review by matobet — CHANGES_REQUESTED** [review]

**Comment by matobet** [comment]
@bond95 please rebase and resolve conflicts

**Comment by mareklibra** [comment]
Rebase is needed because of https://github.com/oVirt/ovirt-ui-components/pull/35.
Please consider moving the components to ovirt-web-ui.

**Comment by bond95** [comment]
It is not give a sense to rebase, because all changes was rebased to web-ui.

**Pull request closed without merging**